python-avd/pyavd/_schema/__init__.py

```diff
@@ -1,3 +1,6 @@
 # Copyright (c) 2023-2024 Arista Networks, Inc.
 # Use of this source code is governed by the Apache License 2.0
 # that can be found in the LICENSE file.
+from .build_schemas import check_schemas
+
+check_schemas()
```

Comment:
I think this could be done as part of the schema store code instead. The schema store will be used to read the schemas when needed. Notice there are two variants of the schema store. One for pyavd and one for schema_tooling.

Reply:
same this will be going